app/javascript/dashboard/routes/dashboard/conversation/contact/ConversationForm.vue

```diff
@@ -259,6 +293,14 @@ export default {
       }
       return this.inboxes.length === 0 && !this.uiFlags.isFetchingInboxes;
     },
+    isSignatureEnabledForInbox() {
+      return this.isAnEmailInbox && this.sendWithSignature;
```

Comment:
Are we enabling it only for email?

Reply:
Yeah, I think so. The linear issue is based on adding a signature for a new email conversation. If we are planning, I can create a separate PR.